mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -68,6 +68,10 @@ HVR:
 		Speed: 99
 		Locomotor: hover
 		TerrainOrientationAdjustmentMargin: -1
+		TurnsWhileMoving: true
+		CanMoveBackward: true
```

Comment:
It doesn't really make sense to have backwards movement on a unit that is unrestricted by rotation

Reply:
That is for a visual effect of delay turning effect on hover unit while they are moving backward, TS style hover unit. We have discussed that in https://github.com/OpenRA/OpenRA/pull/20428#issuecomment-1300146531

Reply:
To make comparsion I will post 2 pics:

1. With backward
![withbackward](https://user-images.githubusercontent.com/13763394/230599598-976078bb-64d4-40d4-aeea-b156fbb784f7.gif)

2. Without backward
![withoutbackward](https://user-images.githubusercontent.com/13763394/230599627-5aff44ef-254b-4315-88c0-e3f8442aa188.gif)

Reply:
Removed, but I think add it will be cooler.